docs/scratch_orgs.rst

```diff
@@ -1,38 +1,172 @@
-Manage scratch orgs
+Manage Scratch Orgs
 ===================
 
-What is an org?
----------------
+Throughout this section, we'll focus on scratch org management in the CumulusCI keychain. To learn about managing persistent orgs, such as sandboxes, production orgs, and packaging orgs, read TODO: reference "Connecting persistent orgs".
+
+What is an Org in CumulusCI?
+----------------------------
+
+CumulusCI takes a different approach to creating and using scratch orgs that aims to make the process easier, more portable, and more repeatable. An org in CumulusCI's keychain starts out as a named configuration, tailored for a specific purpose within the lifecycle of the project. The scratch org is only actually generated the first time you use the scratch org from the project keychain - and once it's expired or been deleted, a new one can easily be created with the same configuration.
 
-Set up the Salesforce CLI
+CumulusCI offers tools that make it easy to discover predefined org configurations, create scratch orgs based on those configurations, and define new orgs and new configurations.
+
+Set Up the Salesforce CLI
 -------------------------
 
-Predefined orgs
+If you haven't already set up the Salesforce CLI, you need to take care of a few steps. For a detailed introduction to setting up Salesforce CLI and Visual Studio Code to work with CumulusCI, we recommend completing `Build Applications with CumulusCI <https://trailhead.salesforce.com/en/content/learn/trails/build-applications-with-cumulusci>`_ on Trailhead.
+
+1. `Install the Salesforce CLI <https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_install_cli.htm>`_
+2. `Enable Dev Hub in Your Org <https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_enable_devhub.htm>`_
+3. `Connect SFDX to Your Dev Hub Org <https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_auth_web_flow.htm>`_ (be sure to use the ``--setdefaultdevhubusername`` option).
+
+If you already have the ``sfdx`` command installed, have connected to your Dev Hub, and have set the ``defaultdevhubusername`` config setting (use ``sfdx force:config:list`` to verify), you're ready to start using CumulusCI with Salesforce CLI.
```

Comment:
Probably better to verify using `force:org:list` and looking for the `(D)` indicator. `force:config:list` can be confusing because of the distinction between global- and project-level settings. It's also [deprecated](https://github.com/forcedotcom/cli/blob/master/releasenotes/README.md#4980-august-27-2020)